lib/defaults.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 module.exports = function () {
   return Object.seal({
+    'host': '',
```

Comment:
Could this default to 'localhost'?
Reason being I think that's the primary use case, and it wouldn't break any existing usage.

Otherwise, I think everything else looks good, thanks!